select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -115,66 +110,6 @@ public AbstractSelect2Choice(String id, IModel<M> model, ChoiceProvider<T> provi
 	}
 
 	/**
-	 * Construct.
-	 *
-	 * @param id
-	 *            markup id
-	 * @param model
-	 *            model for select
-	 * @param collection
-	 *            list options for select
-	 */
-	public AbstractSelect2Choice(String id, IModel<M> model, List<T> collection)
```

Comment:
Why there should be no possibility to create Select2Choice with "pre-selected" options?